rfcs/0018-glides-trainsheet-events.md

```diff
@@ -102,110 +110,119 @@ A [GTFS Time](https://github.com/google/transit/blob/master/gtfs/spec/en/referen
-- `arrivalTime` (Time): when the vehicle is scheduled to arrive at the given location
-- `endLocation` (Location): where the trip is ending
+- `startLocation` (Location): where the trip is scheduled to start
+- `endLocation` (Location): where the trip is scheduled to end
+- `departureTime` (Time): when the trip is scheduled to depart `startLocation`
+- `arrivalTime` (Time): when the trip is scheduled to arrive at `endLocation`
+
 ```json
-`{"serviceDate": "2023-01-20", "arrivalTime": "09:10:11", "endLocation": "id:yard-inner"}
+{
+  "serviceDate": "2023-01-20",
+  "startLocation": {"gtfsId": "place-mdftf"},
+  "endLocation": {"gtfsId": "place-heath"},
+  "departureTime": "4:47:00",
+  "arrivalTime": "5:34:00"
+}
 ```
-
-### Trip Type
-When adding a trip, there are three kinds of trips which can be added:
-- `"arrival"`: a trip arriving to `endLocation` at `time`
-- `"departure"`: a trip departing from `startLocation` at `time`
+*Notes*:
+- The added/scheduled distinction here is internal to Glides, and may not align with what is in other data sources. For example, an Added Trip in Glides may still correspond to a scheduled GTFS trip, and a Scheduled Trip may not appear in GTFS (due to track closures, for example)
 
 ## Events
-### Operator signed in
-At the start of their shift, operators need to confirm that they are fit-for-duty and do not have any electronic devices. They do this by either physically signing a paper trainsheet, or by typing their badge number into Glides. This event represent either type of signin, but may not be generated in the case where an operator only signs into the paper trainsheet.
-
-Event type: `com.mbta.ctd.glides.operator-signed-in`
-Fields in the event:
-- `author` (Author): the inspector who signed in the operator
-- `operator` (Operator): the operator who signed in
-- `signedInAt` (RFC3999 timestamp): the time at which they signed in (separate from the `time` of the event)
-
 ### Batched
 If multiple events are generated as a part of the same author action, then they can be combined into a single Batched event. This ensures that all of the events are kept together throughout the data pipeline.
 
-Event type: `com.mbta.ctd.glides.batched`
+Event type: `com.mbta.ctd.glides.batched.v1`
 Fields in the event:
-- `author` (Author): the inspector who performed the events
-- `events` (array of `{"type": string, "data": JSON}`): each element in the array is a CloudEvent including only the `type` and `data` fields. See [Examples][#examples] for an example.
+- `author` (Author, optional): the inspector who performed the events, if it is the same for all the events in the `events` array. 
+- `events` (array of `{"type": string, "data": JSON}`): each element in the array is a CloudEvent including only the `type` and `data` field, and the `data` fields of the individual events do not need to re-include the `author` field if it is the same as the top-level `author`. See [Examples](#examples) for an example.
```

Comment:
I'd go further and say that it should not re-include the author. If there's an event with a separate author, it should be a separate event (though can do higher-level kinesis batching).